scout/adapter/mongo/omim.py

```diff
@@ -87,16 +87,12 @@ def omim_to_genes(self, omim_obj: dict) -> List[dict]:
 
     def disease_term(
         self,
-        disease_identifier: Union[str, int],
```

Comment:
Don't forget to remove Union from the imports, line 3 of this file, as also suggested by Vulture